src/pretalx/cfp/templates/cfp/event/user_submission_edit.html

```diff
@@ -8,6 +8,26 @@
         {% trans "Current state of your submission:" %}
         {% include "cfp/event/fragment_state.html" with state=submission.state %}
     </p>
+    <h5>
+        <form class="add-speaker" action="{{ submission.urls.invite }}">
+            <div class="form-group form-inline">
+                {% blocktrans trimmed count count=submission.speakers.count %}
+                    Speaker:
+                    {% plural %}
+                    Speakers:
+                {% endblocktrans %}
+                {{ submission.display_speaker_names }},
+                <div class="input-group">
+                    <input name="email" class="form-control form-control-sm" placeholder="another.speaker@mail.org">
```

Comment:
The form doesn't complain if the user inputs something else, like a speaker name. The next page shows the data entered here as "Speaker", not as "Speaker E-Mail". Maybe display a yellow warning thingy on the next page.

Reply:
Fixed both.